envconfig_test.go

```diff
@@ -204,6 +204,21 @@ func TestNonBlankDefaultVar(t *testing.T) {
 	}
 }
 
+func TestExplicitBlankDefaultVar(t *testing.T) {
+	var s Specification
+	os.Clearenv()
+	os.Setenv("ENV_CONFIG_DEFAULTVAR", "")
+	os.Setenv("ENV_CONFIG_REQUIREDVAR", "requiredvalue")
```

Comment:
can we just set `""` for `RequiredVar` as well and test that it satisfies the `required` constraint?